docs/docs/services/qix-engine/access-control.md

```diff
@@ -54,64 +143,89 @@ Change owner | Change the owner of a resource.
 Change role  | Change user role.
 Export data  | Export data from an object.
```

Comment:
- I think these actions names should be displayed and named as how they are used in code. Like
`create` (not Create), `export` (not Export data) and so on.
- Is the list complete? Looking at some of the examples we have, there `reload` and `import` as well. Maybe other missing too?

Reply:
Agree about syntax, don't know about completeness.